src/js/events/ChronosActions.js

```diff
@@ -19,9 +20,20 @@ const ChronosActions = {
             {name: 'embed', value: 'activeJobs'},
           ],
           success: function (response) {
+            let data;
+            try {
+              data = ChronosUtil.parseJobs(response);
+            } catch (error) {
+              AppDispatcher.handleServerAction({
+                type: REQUEST_CHRONOS_JOBS_ERROR,
+                data: error
+              });
+              reject();
+            }
+
             AppDispatcher.handleServerAction({
```

Comment:
Let's move the whole `handleServerAction` and `resolve` part into the `try` block or add a return to the `catch` block. Otherwise we're always dispatching the success event.